chart/templates/malcolm_configmaps.yaml

```diff
@@ -358,6 +358,15 @@ metadata:
   name: zeek-env
 ---
 apiVersion: v1
+kind: ConfigMap
```

Comment:
Would not include the configmap at all if kafka enabled is set to false.

Reply:
I will look into it.  However, having the extra envs doesn't do any harm.  They just won't be used.

Reply:
Done.